config/webhook/manifests.yaml

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
I'm puzzled how it this change related to injecting CA cert, can you please provide me more insights?

Reply:
It should specify how a webhook should be served, I need to confirm this information however. Will post it here.

Reply:
Maybe you can have this change in a separate commit in order to differentiate from removing dependency on the CA cert?

Reply:
Commit added.

Reply:
Thanks!